spec/inject_method_spec.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/inject_method' 
+
+describe My_Array do
+
+	it 'is expected to give the same result for #challenge method and inject method' do
+		subject = My_Array.new
+		expect(subject.challenge(0) {|x, y| x + y}).to eq subject.inject(0) {|x, y| x + y}
```

Comment:
Line is too long. [84/80]<br>Space between { and | missing.<br>Space missing inside }.<br>Tab detected.